spec/transaction_search_shared_examples.rb

```diff
@@ -11,8 +11,9 @@
       let(:params) { super().merge(account_owners: [3, 4]) }
 
       it "applies the parameter" do
-        expect(assigns(:order_details).where_values)
-          .to be_include("account_users.user_id in ('3','4')")
+        where_values = assigns(:order_details).where_values.reject { |v| v.is_a?(String) }
+        expect(where_values.map(&:to_sql))
+          .to include("`account_users`.`user_id` IN (3, 4)")
       end
```

Comment:
Yep, oracle doesn't like this. It was working before before because we were setting a direct string so it was consistent across databases. I'm really not sure what the right fix is for this. Any ideas?